layer/nrlf/core/validators.py

```diff
@@ -116,6 +116,8 @@ def validate(self, data: Dict[str, Any] | DocumentReference):
             self._validate_relates_to(resource)
             self._validate_ssp_asid(resource)
             self._validate_category(resource)
+            if resource.content[0].extension:
```

Comment:
Content is mandatory right? Do we check it somewhere? Otherwise if content is empty at this point, it would be a problem.

Reply:
we do validate the content exists against the model first i believe in our endpoints and also in the validate ssp asid method we check content stuff at that point too

Reply:
cos in the validate method we parse it into that document reference object from the pydantic model and if content isnt there it raises an exception as its mandatory in the model

Reply:
if there is a unit test covering the case where the content is empty before this point, then this is ok.